src/qt/pivx/sendmultirow.cpp

```diff
@@ -71,6 +76,40 @@ void SendMultiRow::amountChanged(const QString& amount)
     Q_EMIT onValueChanged();
 }
 
+void SendMultiRow::onMemoClicked()
+{
+    launchMemoDialog();
+}
+
+bool SendMultiRow::launchMemoDialog()
+{
+    showHideOp(true);
+    SendMemoDialog* dialog = new SendMemoDialog(window, walletModel);
+    dialog->setMemo(recipient.message);
+    bool ret = false;
+    if (openDialogWithOpaqueBackgroundY(dialog, window, 3, 5)) {
+        QString memo = dialog->getMemo();
+        if (IsValidUTF8(memo.toStdString())) {
```

Comment:
utf-8 validation shouldn't be needed (https://doc.qt.io/qt-5/qstring.html#toStdString):
> [QString::toStdString()] The Unicode data is converted into 8-bit characters using the toUtf8() function.